core/src/save/region/mod.rs

```diff
@@ -344,6 +351,18 @@ fn read_section_into_chunk(section: &LevelSection, chunk: &mut Chunk) -> Result<
 }
 
 fn chunk_to_chunk_root(chunk: &Chunk, entities: Vec<EntityData>) -> ChunkRoot {
+    let heightmaps: Vec<i64> = chunk
+        .heightmaps()
+        .iter()
+        .map(|map| {
+            (map.motion_blocking() as i64)
```

Comment:
⛳️ 

Perhaps there should be `From`/`Into` Implementations for HeightMap -> i64 and i64 -> HeightMap

Reply:
Same thought, however, would I not be moving serlization details into heightmap?

Reply:
However maybe we want to be able to modify the generated height at some point, and it should be up to the user if she/he has the need?